client/src/epics/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Here's the doc on the rule:
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md

Umm, they can be exported as the default. the only problem is is that you can get in a double export scenario:
```
export const getSomething = () => {

}

export default getSomething;
```

If you do a "namespace import" in another file like this:
```
import * as SomethingEpics from './something'
```
The `SomethingEpics` object would have it in there twice, which is normally fine, but if you do `Object.values(SomethingEpics)` like we are in `epics/index.js` then you'd get it twice. This results in the epic being run more than once (which took me a while to figure out).  As far as I know there isn't an eslint rule (that I can find) that prevents double exports.

I generally really like the default export rule, especially for components.

Side note: honestly I could go a little further and make it _only_ do default exports. That would have us end up with a single function/class per file (kinda like PHP's single class per file).

At a minimum, we could fix this by just not doing namespace imports and implementing
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-namespace.md
for the whole project. I don't think this would be a bad thing. It would make some files (like `epics/index.js`) more verbose, but perhaps that's not a bad thing. I think this is the best solution, what do you think?